config/pipeline.php

```diff
@@ -46,12 +46,14 @@
     $app->pipe(PathBasedRoutingMiddleware::class);
```

Comment:
Maybe we can explain here that at this point the RouteResult::class is injected. It took me 20 minutes yesterday to figure out why a middleware wasn't working :)

Reply:
Done!